packages/gafl-webapp-service/src/services/address-lookup/address-lookup-service.js

```diff
@@ -44,7 +46,25 @@ export default async (premises, postcode) => {
       locality: r.locality,
       town: r.city,
       postcode: r.postcode,
-      country: r.country
+      country: r.country,
+      address_full: formatAddress(r.premises, r.street_address, r.locality, r.city, r.postcode)
```

Comment:
rather than doing this, modify the address property to be:
```
address: `${r.address.replace(r.postcode, '').toLowerCase()}${r.postcode}`
```
transforming the address, apart from the postcode, to be all lowercase allows you to use the css property text-transform, as the capitalise value will transform lower-case text, but not upper case text